cranelift/codegen/src/legalizer/heap.rs

```diff
@@ -66,19 +66,14 @@ fn dynamic_addr(
 
     // Start with the bounds check. Trap if `offset + access_size > bound`.
     let bound = pos.ins().global_value(offset_ty, bound_gv);
-    let oob;
-    if access_size == 1 {
+    let cmp = if access_size == 1 {
```

Comment:
nit: for symmetry with the other function, can you use a destructuring assignment here too? `let (cc, offset, bound) = ...`

Reply:
Done!